cypress/e2e/spaces/spaces.spec.ts

```diff
@@ -32,7 +33,7 @@ function openSpaceContextMenu(spaceName: string): Chainable<JQuery> {
     return cy.get(".mx_SpacePanel_contextMenu");
 }
 
-function spaceCreateOptions(spaceName: string): ICreateRoomOpts {
+function spaceCreateOptions(spaceName: string, roomIds: string[] = []): ICreateRoomOpts {
```

Comment:
Is this change related to the rest or a small extra fix?

Reply:
This is extending a test utility to be able to create a space with a room in it to streamline test fixture setup